scripts/deploy.py

```diff
@@ -206,5 +231,19 @@ def _execute_deployment():
     print 'Done!'
 
 
+def _store_cache_slug():
+    """Changes the cache slug in cache_slug.yaml"""
+    cache_slug = utils.get_unique_id()
+
+    # Change the cache slug in cache_slug.yaml.
+    f = open('cache_slug.yaml', 'r')
+    content = f.read()
+    os.remove('cache_slug.yaml')
+    content = content.replace('default', cache_slug)
+    d = open('cache_slug.yaml', 'w+')
+    d.write(content)
+
+
 if __name__ == '__main__':
+    _store_cache_slug()
```

Comment:
what do you think about _update_cache_slug() ?
Also, isn't this call needs to be as part of _execute_deployment() ?